stglib/hobo.py

```diff
@@ -1,12 +1,14 @@
+import csv
+
 import numpy as np
 import pandas as pd
 import xarray as xr
 
-from .core import utils
+from .core import qaqc, utils
 
 
 def read_hobo(
-    filnam, skiprows=1, skipfooter=0, names=["#", "datetime", "abspres_kPa", "temp_C"]
+    filnam, skiprows=1, skipfooter=0, names=["#", "DateTime", "AbsPres_kPa", "Temp_C"]
```

Comment:
Has the capitalization formatting changed with recent Hoboware updates? If so, do you know when? The all-lowercase used to work.

Reply:
Note sure. should we try to handle both? All the files I was working with had the upper case letters for first letter of keywords in the name. I guess one way to do it would be to force to lower case using lower().